tests/ci/docker_images/build_cryptofuzz_modules.sh

```diff
@@ -60,3 +57,6 @@ env CRYPTOFUZZ_DICT `realpath cryptofuzz-dict.txt`
 env CFLAGS "$CFLAGS"
 env CXXFLAGS "$CXXFLAGS"
 env CRYPTOFUZZ_SRC "$CRYPTOFUZZ_SRC"
+
+# Cryptofuzz builds it's modules into $CRYPTOFUZZ_SRC/modules that includes everything it needs, this saves a substantial amount of space
```

Comment:
Nit: builds "its" modules
Also, may be change ", this saves" -> "; deleting it saves" (if I understood correctly)